proto/lbm/zkauth/v1/params.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+package lbm.zkauth.v1;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/Finschia/finschia-sdk/x/zkauth/types";
+
+message Params {
+    option (gogoproto.goproto_stringer) = false;
+    
+    // FetchIntervals is the interval at which the node fetches the oauth_pub_key.
+    uint64 fetch_intervals = 1;
+}
```

Comment:
I think it is not necessary that add `fetch_intervals` into the genesis. I think it's nice to change to config setting. How about this?

Reply:
Ok, I removed it from genesis.